Dockerfile

```diff
@@ -3,6 +3,6 @@ FROM praekeltfoundation/django-bootstrap:py3.6
 COPY . /app
 RUN pip install -e .
 
-ENV DJANGO_SETTINGS_MODULE "rp_sidekick.settings"
-RUN python manage.py collectstatic --noinput
+ENV DJANGO_SETTINGS_MODULE "config.settings.production"
+RUN SECRET_KEY=placeholder ALLOWED_HOSTS=placeholder RAPIDPRO_API_KEY=placeholder python manage.py collectstatic --noinput
```

Comment:
What is the `RAPIDPRO_API_KEY` variable for?

Reply:
Oops, copy/paste error. Thanks!